src/test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol

```diff
@@ -0,0 +1,649 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "../utils/MockAVSDeployer.sol";
+
+contract BLSRegistryCoordinatorWithIndicesUnit is MockAVSDeployer {
+    using BN254 for BN254.G1Point;
+
+    uint8 maxQuorumsToRegisterFor = 4;
+
+    event OperatorSocketUpdate(bytes32 operatorId, string socket);
+
+    /// @notice emitted whenever the stake of `operator` is updated
+    event StakeUpdate(
+        bytes32 indexed operatorId,
+        uint8 quorumNumber,
+        uint96 stake
+    );
+
+    // Emitted when a new operator pubkey is registered for a set of quorums
+    event PubkeyAddedToQuorums(
+        address operator,
+        bytes quorumNumbers
+    );
+
+    // Emitted when an operator pubkey is removed from a set of quorums
+    event PubkeyRemovedFromQuorums(
+        address operator, 
+        bytes quorumNumbers
+    );
```

Comment:
should these be named 'PubkeyXYZ' if they emit the `operator` specifically, rather than the pubkey?
Also seems like the `operator` should be indexed

Reply:
any suggestions?

Reply:
Literally just `OperatorAddedToQuorums` and `OperatorRemovedFromQuorums` instead?
I'm also unsure on our off-chain usage, maybe actually having the pubkey in these would be better?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/88/commits/f2f25d245b79e36f7fbb7551fd1da18f4f30f0ba